src/vmm/src/vstate/vcpu/mod.rs

```diff
@@ -38,6 +39,8 @@ pub(crate) mod x86_64;
 
 #[cfg(target_arch = "aarch64")]
 pub(crate) use aarch64::{Error as VcpuError, *};
+#[cfg(test)]
+use std::sync::Mutex;
```

Comment:
nit: this should move up an be with the other stdlib `use`s (around line 10)

Reply:
done